lib/signet/vm.ex

```diff
@@ -870,7 +886,7 @@ defmodule Signet.VM do
   end
 
   def exec(code, calldata, opts) when is_list(code) do
-    run_code(Context.init_from(code, Keyword.get(opts, :ffis, %{})), %Input{
+    run_code(Context.init_from(code, Keyword.get(opts, :ffis, @builtin_ffis)), %Input{
```

Comment:
Shouldn't we combine the client provided FFIs with the built-in ones? Since these are framework-level FFIs, I think they should always be accessible to the client.

Reply:
yeah, I thought this was a merge, you're right.